superx/tests/test_livesearch.py

```diff
@@ -15,17 +15,20 @@ def test_livesearch(client):
     item_input3 = {'input': "     "}
     item_input4 = {'input': ""}
 
-    do_it(client, item_input1)
-    do_it(client, item_input2)
-    do_it(client, item_input3)
-    do_it(client, item_input4)
+    city_input = {'city': 'ירושלים'}
 
+    do_it(client, item_input1, city_input)
+    do_it(client, item_input2, city_input)
+    do_it(client, item_input3, city_input)
+    do_it(client, item_input4, city_input)
 
-def do_it(client, item_input):
+
+def do_it(client, item_input, city_input):
```

Comment:
the function name do_it isn't very descriptive. 
Maybe it can be changed to execute_test or similar